spec/features/crystal_sets/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+#spec/features/crystal_sets/index_spec
+require 'rails_helper'
+# As a visitor
+# When I visit the parent index,
+# I see that records are ordered by most recently created first
+# And next to each of the records I see when it was created
+
+RSpec.describe 'crystal set index' do
+  #gem orderly - checks display order (expect certain text appears before other text)
+  #within blocks 
+  # expect(model_name.column_name).to appear_before(whatever)
+  # https://github.com/jmondo/orderly
+
+  it 'can see all crystal sets and their creation date' do 
+    set = CrystalSet.create!(collection_name: "Raw Crystal Chunks - 28pc", limited_edition: false, inventory: 1000)
+
+    visit '/crystal_sets'
+
+    expect(page).to have_content(set.collection_name)
+    expect(page).to have_content(set.limited_edition)
+    expect(page).to have_content(set.created_at.strftime('%m/%d/%y'))
+  end
+
+  it 'shows crystal sets in order of date created' do
+    visit '/crystal_sets'
+
+    expect("Intuitively Chosen Raw Crystals Collection").to appear_before("Raw Crystal Chunks - 28pc Collection", only_text: true)
```

Comment:
I would check this assertion by printing the whole time stamp. I found that spec was running so fast that all of the sets had the same creation time. You can add a `created_at` to the create! command to be sure that they are appearing in the correct order. The `save and open page` command helped me a lot here.

Reply:
Ahh that's a really good point! I'm also a little confused on the `save_and_open_page` command. I thought it was something akin to using pry in your test just to make sure that your test runs properly. Do you need to keep it in?